.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
So it depends if you want to work on other things first because working on the workflow script. 
(If you wanna do later, we can file an issue for now.)